nucleus/deploy/find_packages.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python
```

Comment:
Copied from `models/demos/find_packages.py` with some changes

Reply:
Curious, what changes?

Reply:
mostly lint changes, I also changed a line to `if module_name == "nucleus"` somewhere, and had some functions at the end return values instead of just printing them out